pom.xml

```diff
@@ -52,6 +52,12 @@
 
     <!-- For surefire and failsafe to be compatible with jacoco -->
     <argLine>-Xmx1024m -XX:MaxPermSize=512m</argLine>
+    
+    <!-- Sonar properties -->
+    <sonar.projectKey>Meeds-io_social</sonar.projectKey>
```

Comment:
we can declare less variables, by adding in parent pom properties : 
`
<sonar.projectKey>${project.groupId}</sonar.projectKey>
<sonar.host.url>https://sonarcloud.io</sonar.host.url>
<sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
`

Reply:
ok, I will do that.

Reply:
done